docs/known_issues.rst

```diff
@@ -29,59 +29,15 @@ Known Deficiencies
 Quantities Lose Their Units with Some Operations
 ------------------------------------------------
 
-Quantities are subclassed from NumPy's `~numpy.ndarray` and in some NumPy
-operations (and in SciPy operations using NumPy internally) the subclass is
-ignored, which means that either a plain array is returned, or a
-`~astropy.units.quantity.Quantity` without units.
-E.g., prior to astropy 4.0 and numpy 1.17::
+Quantities are subclassed from ``numpy``'s `~numpy.ndarray` and while we have
+ensured that ``numpy`` functions will work well with them, they do not always
+work in functions from ``scipy`` or other packages that use ``numpy``
+internally, but ignore the subclass. Furthermore, at a few places in ``numpy``
+itself we cannot control the behaviour. For instance, care must be taken when
+setting array slices using Quantities::
 
     >>> import astropy.units as u
     >>> import numpy as np
-    >>> q = u.Quantity(np.arange(10.), u.m)
-    >>> np.dot(q,q) # doctest: +SKIP
-    285.0
-    >>> np.hstack((q,q)) # doctest: +SKIP
-    <Quantity [0., 1., 2., 3., 4., 5., 6., 7., 8., 9., 0., 1., 2., 3., 4., 5.,
-               6., 7., 8., 9.] (Unit not initialised)>
-
-And for all versions::
-
-    >>> ratio = (3600 * u.s) / (1 * u.h)
-    >>> ratio # doctest: +FLOAT_CMP
-    <Quantity 3600. s / h>
-    >>> np.array(ratio) # doctest: +FLOAT_CMP
-    array(3600.)
-    >>> np.array([ratio]) # doctest: +FLOAT_CMP
-    array([1.])
-
-Workarounds are available for some cases. For the above::
-
-    >>> q.dot(q) # doctest: +FLOAT_CMP
-    <Quantity 285. m2>
-
-    >>> np.array(ratio.to(u.dimensionless_unscaled)) # doctest: +FLOAT_CMP
-    array(1.)
-
-    >>> u.Quantity([q, q]).flatten() # doctest: +FLOAT_CMP
-    <Quantity [0., 1., 2., 3., 4., 5., 6., 7., 8., 9., 0., 1., 2., 3., 4., 5.,
-               6., 7., 8., 9.] m>
-
-An incomplete list of specific functions which are known to exhibit
-this behavior (prior to astropy 4.0 and numpy 1.17) follows:
-
-* `numpy.dot`
-* `numpy.hstack`, `numpy.vstack`, ``numpy.c_``, ``numpy.r_``, `numpy.append`
-* `numpy.where`
-* `numpy.choose`
-* `numpy.vectorize`
-* pandas DataFrame(s)
-
-
-See: https://github.com/astropy/astropy/issues/1274
-
-
-Care must be taken when setting array slices using Quantities::
-
```

Comment:
I was actually wondering the other day if these "known issues" were still issues.  I assume we've confirmed that these are no longer known issues?

Reply:
The only `numpy` example that is still relevant is the one with `np.array()`, but that was safe to delete because there is an example about this in the Quantities Lose Their Units When Broadcasted subsection.

Reply:
yes, it is nice how many issues are no longer a problem!